app/assets/stylesheets/custom.scss

```diff
@@ -1,4 +1,44 @@
+//  home page
 .home-search {
   vertical-align:middle;
   margin-bottom:0px;
+  margin-top: 0px;
+  width: 700px;
+  margin: 0 auto;
 }
+
+.navbar-header {
+  margin-top: 10px;
+  margin-left: 110px;
+  font-size: 21px;
+}
+
+.navbar.transparent .navbar-inner {
+	background: transparent;
+	position: absolute;
```

Comment:
Line should be indented with spaces, not tabs<br>Properties should be ordered background, left, position, right, top, z-index